transformer_engine/pytorch/utils.py

```diff
@@ -386,3 +387,12 @@ def nvtx_range_pop(msg: Optional[str] = None) -> None:
 
     # Pop NVTX range
     torch.cuda.nvtx.range_pop()
+
+
+def get_torch_version():
+    """Get PyTorch version from __version__"""
+
+    def get_torch_version_str():
+        return str(torch.__version__)
+
+    return PkgVersion(get_torch_version_str())
```

Comment:
Does this indirection give us anything?
```python
def get_torch_version() -> PkgVersion:
    """Get PyTorch version"""
    return PkgVersion(str(torch.__version__))
```
I'd recommend making some more changes:
```suggestion
@lru_cache(maxsize=None)
def torch_version() -> tuple[int, ...]:
    """Get PyTorch version"""
    return PkgVersion(str(torch.__version__)).release
```
The name is cleaner, tuples are easier to work with than `Version` (so you don't need to deal with the messiness of `importlib` and `packaging`), and caching reduces overhead.